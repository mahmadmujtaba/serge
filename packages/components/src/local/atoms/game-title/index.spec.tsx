/* global it expect */

import React from 'react'
import renderer from 'react-test-renderer'

import GameTitle from './index'

const onTitleClick = (title: string) => {
  window.alert(title)
  return {}
}

describe('GameTitle component:', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <GameTitle initiated={false} title="IMWARC-New" onClick={onTitleClick} />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
