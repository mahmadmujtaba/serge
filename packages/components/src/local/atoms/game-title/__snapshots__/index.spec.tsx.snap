// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameTitle component: renders correctly 1`] = `
<span
  className="wargame-title"
>
  <span
    className="wargame-unstarted-highlight"
  >
    Â 
  </span>
  <span
    onClick={[Function]}
    title="IMWARC-New"
  >
    IMWARC-New
  </span>
</span>
`;
